### ISSUE TITLE
[Feature Suggestion] Additional feature to effect "Snow"

### ISSUE THREAD

**Issue by MrJayGamerTTV** [gta-chaos-mod/ChaosModV#1589]
**Is your feature request related to a problem? Please describe.**
No, it is not related to a problem.

**Describe the solution you'd like**
When the effect "Snow" happens and the player has police after them, I would like to see the police cruisers that spawn around you to chase you, instead of normal cruisers, change them to spawn in the North Yankton police cruisers, as those cruisers fit better into the snowy environment that this effect brings in. At the same time should there be other vehicles that can be brought from North Yankton (civilian vehicles), change it that every vehicle that spawns into the world during this effect will be the ones from North Yankton.

Long story short - When effect "Snow" happens, change the vehicles around you, including police cruisers, to North Yankton vehicles.

**Describe alternatives you've considered**
--

**Additional context**
--
